packages/change-password/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@availity/change-password",
+  "version": "6.5.9",
```

Comment:
Do you mind setting this to `1.0.0`?

Reply:
See [7cfa372](https://github.com/Availity/availity-react/pull/1297/commits/7cfa372ec65c002ecfa0e57c1025207e6bac2d05)